toolkit/tools/scheduler/scheduler.go

```diff
@@ -178,7 +188,7 @@ func cancelBuildsOnSignal(signals chan os.Signal, agent buildagents.BuildAgent)
 
 // buildGraph builds all packages in the dependency graph requested.
 // It will save the resulting graph to outputFile.
-func buildGraph(inputFile, outputFile string, agent buildagents.BuildAgent, workers, buildAttempts int, stopOnFailure, canUseCache bool, packagesToBuild []*pkgjson.PackageVer, packagesNamesToRebuild, ignoredPackages []string) (err error) {
+func buildGraph(inputFile, outputFile string, agent buildagents.BuildAgent, workers, buildAttempts int, stopOnFailure, canUseCache bool, packagesToBuild []*pkgjson.PackageVer, packagesNamesToRebuild, ignoredPackages, reservedFiles []string) (err error) {
```

Comment:
should ignorgedPackages still be listed with "[]string" ?

Reply:
Both are the same type (`[]string`). I think this is fine.